pipe.go

```diff
@@ -73,6 +74,7 @@ type pipe struct {
 	nsubs           *subs                       // pubsub  message subscriptions
 	psubs           *subs                       // pubsub pmessage subscriptions
 	info            map[string]RedisMessage
+	az              string
```

Comment:
Can we remove this? Just relying on the `info` is enough.

Reply:
[908bc94](https://github.com/redis/rueidis/pull/721/commits/908bc940f1ea3372e4e2236d8426184d2f032163)